src/api.ts

```diff
@@ -9,7 +9,8 @@ import {
 import { Bridge } from './bridges'
 import { Exchange, ExchangeAggregator } from './exchanges'
 
-export type Order = 'RECOMMENDED' | 'FASTEST' | 'CHEAPEST' | 'SAFEST'
+export const _Order = ['RECOMMENDED', 'FASTEST', 'CHEAPEST', 'SAFEST'] as const
+export type Order = typeof _Order[number]
```

Comment:
Why do you need this? Also `_` usually indicates a private property so exporting it feels a bit weird I think?

Reply:
We need this for route priority selection in the widget. I made it like this to follow other unions naming in this file. We can rename it if needed. 🙂

Reply:
Orders will be fine.

Reply:
Yes, already been published. :)